src/d/actor/d_a_shutter.cpp

```diff
@@ -5,80 +5,284 @@
 
 #include "d/actor/d_a_shutter.h"
 #include "d/d_procname.h"
+#include "d/d_com_inf_game.h"
+
+const float daShutter_c::m_max_speed[2] = {3.0f, 3.0f};
+const float daShutter_c::m_min_speed[2] = {1.0f, 1.0f};
+const float daShutter_c::m_move_len[2] = {200.0f, 84.0f};
+const float daShutter_c::m_width[2] = {220.0f, 82.0f};
+const float daShutter_c::m_height[2] = {200.0f, 100.0f};
+const s16 daShutter_c::m_bdlidx[2] = {4, 4};
+const s16 daShutter_c::m_dzbidx[2] = {7, 7};
```

Comment:
These are file indexes used to load files, you can find enums for them in the headers `res_htobi1.h` (for [0]) and `res_htobi2.h` (for [1])